### PR TITLE
Firma 348 small fixes

### PR DESCRIPTION
**Pull request by SergioPH**

### CODE REVIEW

File: Firma_iOS/Supporting Files/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -74,7 +76,7 @@
...
+"about_description_label" = "\"Cliente @firma for Apple iOS\" v%@\n2019 Gobierno de España\n\nCliente @firma for Apple iOS is open source; is available for redistribution and/or modification using the EUPL license: \"European Software License 1.1\", the complete text can be found in the next Web: http://ec.europa.eu/idabc/eupl.html\n\nThe source code of this software, as well as all the components used in this project are available to download freely from: http://forja-ctt.administracionelectronica.gob.es\n\nThe project Cliente @firma is a cooperative project, and if you want to participate on it, you shouldn't doubt about joining the community, available in the next Web:\n\nhttps://administracionelectronica.gob.es/ctt/clienteafirma/mas#.Xxp7T0l7mhc";
```
Comment (ggdiez) [inline Firma_iOS/Supporting Files/en.lproj/Localizable.strings:79]: 2020?

File: Firma_iOS/Supporting Files/es.lproj/Localizable.strings [hunk 2]
```diff
@@ -74,7 +76,7 @@
...
+"about_description_label" = "\"Cliente @firma para Apple iOS\" v%@\n2019 Gobierno de España\n\nEl Cliente @firma para Apple iOS es Software Libre; puedes redistribuirlo y/o modificarlo bajo los términos de la siguiente licencia: \"European Software License 1.1\", cuyo texto completo puede consultarse en la siguiente página Web: http://ec.europa.eu/idabc/eupl.html\n\nEl código fuente de este programa, así como del resto de los componentes del proyecto se pueden descargar libremente desde: http://forja-ctt.administracionelectronica.gob.es\n\nEl proyecto Cliente @firma es un proyecto colaborativo, si desea participar en él, no dude en unirse a la comunidad, accesible desde la siguiente página Web:\n\nhttps://administracionelectronica.gob.es/ctt/clienteafirma/mas#.Xxp7T0l7mhc";
```
Comment (ggdiez) [inline Firma_iOS/Supporting Files/es.lproj/Localizable.strings:79]: 2020?

Comment (SergioPH) [reply Firma_iOS/Supporting Files/es.lproj/Localizable.strings]: Done!

File: Firma_iOS/Supporting Files/en.lproj/Localizable.strings (hunk 1 above)
Comment (SergioPH) [reply Firma_iOS/Supporting Files/en.lproj/Localizable.strings]: Done!

**Review by ggdiez — APPROVED** [review]

**Pull request merged**